package.json

```diff
@@ -19,15 +19,20 @@
         {
             "name": "Travis Kosarek",
             "url": "https://github.com/traviskosarek"
+        },
+        {
+            "name": "Caleb Cartwright",
+            "url": "https://github.com/calebcartwright"
         }
     ],
-    "license": "ISC",
+    "license": "MIT",
```

Comment:
I know I am being pedantic here since these are essentially the same types of license, but changing it to MIT here to be consistent with all other license references in the repo

Reply:
Oops, good catch. I didn't see that the license type was wrong there.